### PR TITLE
Feature/styling

### PR DESCRIPTION
**Pull request by JBerlin84**
Satt mest och testa lite styling och siduppbyggnad, men nu har vi en förstasida på sidan.

### CODE REVIEW

File: frontend/web/src/app/components/home/home.component.html [hunk 1]
```diff
@@ -1,3 +1,9 @@
-<p>
-  home works!
-</p>
+<div class="container" id="wrapper">
+  <div class="logo">
+      <img src="http://resources.mynewsdesk.com/image/upload/t_open_graph_image/y2pmtfolvlimvq6ajvkv.jpg" />
```
Comment (chraker) [inline frontend/web/src/app/components/home/home.component.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: frontend/web/src/app/components/home/home.component.html [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+  </div>
+  <h2>In Cooperation With</h2>
+  <div class="logo">
+      <img src="http://www.chalmers.se/SiteCollectionImages/Logotyper/Chalmers%20logotyp/AvancezChalmersU_black_right.png" />
```
Comment (chraker) [inline frontend/web/src/app/components/home/home.component.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

**Comment by chraker** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by mlonn — APPROVED** [review]
looks good

**Pull request merged**